doc/source/examples/extended_examples/hpc/ml_ga_beam.py

```diff
@@ -0,0 +1,279 @@
+    print("=============")
+
+
+# Fitness function
+
+
+def fitness_func(ga_instance, solution, solution_idx):
+    # Query a free MAPDL instance
+    mapdl, i = pool.next_available(return_index=True)
+    mapdl.locked = True
+    mapdl._busy = True
+
+    # Perform chromosome simulation
+    model_output = calculate_beam(mapdl, solution)
+
+    # Releasing MAPDL instance
+    mapdl.locked = False
+    mapdl._busy = False
+
+    # Calculate errors and criteria
+    error_ = calculate_error(model_output)
+    fitness_criteria = calculate_fitness_criteria(model_output)
+
+    # Print at each chromosome solution
+    # Print the port to observe how the GA is using all MAPDL instances
+    print(
+        f"MAPDL instance {i}(port: {mapdl.port})\tInput: {solution[0]:0.1f}\tOutputs: {model_output:0.7f}\tError: {error_:0.3f}%\tFitness criteria: {fitness_criteria:0.6f}"
+    )
+
+    return fitness_criteria
+
+
+def mutation_func(offspring, ga_instance):
+    average = offspring.mean()
+    max_value = offspring.max() - average
+    min_value = offspring.min() - average
+
+    min_value = min([min_value, max_value, -1])
+    max_value = max([min_value, max_value, +1])
+
+    offspring[:, 0] += np.random.randint(min_value, high=max_value, size=offspring.size)
+
+    for i in range(2):
+        random_spring_idx = np.random.choice(range(offspring.shape[1]))
+        sign = np.random.choice([-1, 1])
+        offspring[random_spring_idx, 0] += sign * average * (0.1 * np.random.random())
+
+    return offspring
+
+
+ga_instance = pygad.GA(
+    # Main options
+    sol_per_pop=sol_per_pop,
+    num_generations=num_generations,
+    num_parents_mating=num_parents_mating,
+    num_genes=num_genes,
+    fitness_func=fitness_func,
+    parallel_processing=parallel_processing,
+    random_seed=2,  # to get reproducible results
+    #
+    # Mutation
+    mutation_percent_genes=mutation_percent_genes,
+    mutation_type=mutation_func,
+    mutation_probability=mutation_probability,
+    #
+    # Parents
+    keep_parents=keep_parents,
+    parent_selection_type=parent_selection_type,
+    #
+    # Helpers
+    on_fitness=on_fitness,
+    gene_type=gene_type,
+    init_range_low=init_range_low,
+    init_range_high=init_range_high,
+)
+
+ga_instance.igen = 0  # To count the number of generations
+
+
+# Run simulation
+# ==============
+
+import time
+
+t0 = time.perf_counter()
+
+ga_instance.run()
+
+t1 = time.perf_counter()
+print(f"Time spent (minutes): {(t1-t0)/60}")
+
+
+# Plot convergence
+# ================
+
+ga_instance.plot_fitness(label=["Applied force"], save_dir=os.getcwd())
+
+solution, solution_fitness, solution_idx = ga_instance.best_solution(
+    ga_instance.last_generation_fitness
+)
+
+print(f"Parameters of the best solution : {solution[0]}")
+print(f"Fitness value of the best solution = {solution_fitness}")
+
+
+# Store the model result
+# ======================
+
+from datetime import datetime
+
+# Save the GA instance.
```

Comment:
```suggestion
# Save the GA instance
```
Removing periods as you aren't using them unless there are multiple sentences in the comment.